tests/functional/oauth_query_param_validation.js

```diff
@@ -349,35 +349,6 @@ define([
           client_id: UNTRUSTED_CLIENT_ID,
           scope: UNTRUSTED_SCOPE
         }));
```

Comment:
> There is also a 'verification_redirect': STRING_TYPE.optional() in post-metrics-errors, but do we leave it??

👍 - good catch, I'll remove it. So few people hit used verification_redirect in the past few weeks that I think it's easier to just use it now. Those that did use verification_redirect shouldn't have been anyways according to the testpilot devs.